GUI/Forms/EntityInfoForm.cs

```diff
@@ -98,13 +98,31 @@ public void AddConnection(KVObject connectionData)
             var delay = connectionData.GetFloatProperty("m_flDelay");
             var timesToFire = connectionData.GetInt32Property("m_nTimesToFire");
 
+            string stimesToFire = "";
+
+            switch (timesToFire)
+            {
+                case 1:
+                    stimesToFire = "Only Once";
+                    break;
+                case 2:
+                    stimesToFire = "Only Twice";
+                    break;
+                case >= 3:
+                    stimesToFire = $"Only {timesToFire} Times";
+                    break;
+                default:
```

Comment:
Change this to case -1 so we know it's expected, and any other value would say `-2 times` (shouldn't ever happen anyway).

Reply:
Setting 0 is also infinite times. I am doing this to prevent the occurrence of `Only 0 Times` or `Only -2 Times`.

Reply:
-2 also infinite?

Reply:
Case 0 is fine. I am not confident that -2 would be infinite in the engine.

Reply:
Watch the video in #815, which includes tests for `-2` and `0`.
And this "bug" OR "feature" already exists in the source 1 engine, both of which are infinite.